packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -72,8 +72,7 @@ const CONTENT_MAP: Record<PostType, typeof PostContent> = {
   share: SquadPostContent,
   welcome: SquadPostContent,
   freeform: SquadPostContent,
-  // TODO: remove this once we know what content to use
-  [PostType.VideoYouTube]: null,
+  [PostType.VideoYouTube]: PostContent,
```

Comment:
@sshanzel there might be an issue here. I can open a squad post directly here, and it would render the wrong layout.
![image](https://github.com/dailydotdev/apps/assets/1681525/3d4b35f6-8a16-4714-b059-10b81c2b9c9d)

Reply:
You mean a Squad post for video? If that is the case, it would be handled in a separate file (SquadPostContent) where it decided whether to display the layout for Video.

Reply:
You can see it in the squad I created.
https://app.daily.dev/posts/ce4uyPVi8
https://app.daily.dev/squads/youtubecontent

Reply:
Can you invite me to the Squad? Though, yes, we need to [update something](https://dailydotdev.slack.com/archives/C059GQ5RFP0/p1702440788590979?thread_ts=1702306710.702739&cid=C059GQ5RFP0) in the API side in order to reflect the right layout. Currently, the post would be considered as an ArticlePost rather than VideoYouTube.

Reply:
This is the reason why.

https://github.com/dailydotdev/daily-api/blob/655c6af2fed9e082f8ce7bee3d66fb73dece8035/src/entity/posts/utils.ts#L241

Reply:
Invite [link here](https://dailydotdev.slack.com/archives/C059GQ5RFP0/p1702463442840419)